src/services/auth.service.js

```diff
@@ -1,3 +1,42 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
 
+const userDao = require('../models/user.dao');
+const { validateEmail } = require('../utils/validators');
 
+const signUp = async (email, password) => {
+  validateEmail(email);
```

Comment:
validatePassword 함수를 만들었으니 해당 부분에서도 검증할 수 있도록 import 해와야 될 것 같습니다 승현님~
```suggestion
const { validateEmail, validatePassword } = require('../utils/validators');

const signUp = async (email, password) => {
  validateEmail(email);
  validatePassword(password);
```